src/components/visualizations/table/utils/test/utils.spec.ts

```diff
@@ -0,0 +1,56 @@
+// (C) 2007-2019 GoodData Corporation
+import { ITotalWithData } from "../../../../../interfaces/Totals";
+import { getFooterHeight, getHeaderOffset } from "../utils";
+describe("Table utils - Utils", () => {
```

Comment:
sounds to me like too common name for file... basically it does not say anything what is here... which is wrong